### ISSUE TITLE
[Mano] has a function called device_settings that depends on website structure

### ISSUE THREAD

**Issue by biblicabeebli** [onnela-lab/beiwe-backend#454]
it parses the device settings page looking for enabled data streams (I think). We should make an api endpoint.
(or... not do that because it isn't actually safe to assume those checkboxes correspond to collected data. We can do an exists query on all study collected data, shooouuuld be fastt?)

**Comment by biblicabeebli** [comment]
there is a test test_device_settings that currently does nothing, waiting for this to be implemented.

**Comment by wengthree1cm** [comment]
Update the fetch_study_device_setting function:
- create a new branch called jack/api and create a new function  called fetch_study_settings_test
- write some test use mock response, including error code tests and result test
- test the new function with reael API call and are able to get correct data

**Comment by biblicabeebli** [comment]
@stellenli802 any update?

**Comment by biblicabeebli** [comment]
@wengthree1cm This issue might be done in your current work already, can you confirm?
(Woops, at'd the wrong person there)

**Comment by biblicabeebli** [comment]
@wengthree1cm ping

**Comment by wengthree1cm** [comment]
> [@wengthree1cm](https://github.com/wengthree1cm) ping

yes its done, in my new branch, not sure if it is merged to the main yet. but after this merge it will be in the main anyway

**Comment by biblicabeebli** [comment]
closing because the work is done
